### PR TITLE
build: fraxtal env, rm evm-version

### PR DESCRIPTION
**Pull request by pegahcarter**
Related to #14.

### CODE REVIEW

File: foundry.toml [hunk 1]
```diff
@@ -22,13 +22,11 @@ optimizer = false
 # The number of optimizer runs
 optimizer_runs = 200 
 # use the via-ir compiler flag
-via_ir = false
+via_ir = true
```
Comment (tom2o17) [inline foundry.toml:25]: Personally I am a fan of having this set to `false` for the default profile and swapping to `true` for deploy but defer to you. 

I think for all other repos we default via-ir to `false` and then reset to `true` on `profile.deploy` 

Worry the naive contributor/user will run into issue and get discouraged

**Review by tom2o17 — APPROVED** [review]

File: foundry.toml (hunk 1 above)
Comment (pegahcarter) [reply foundry.toml:25]: Good point!  I did not realize this flag was causing the issue.  I'll revert and merge.

Comment (tom2o17) [reply foundry.toml:25]: Probably could just add to example.env to solve for naive user experience mentioned above

Comment (pegahcarter) [reply foundry.toml]: I'm just going to leave as-is to maintain the old spec as it would require changing the `.env` when deploying.

**Pull request merged**